features/logging/eventrouter.feature

```diff
@@ -0,0 +1,30 @@
+@clusterlogging @commonlogging
+Feature: eventrouter related test
+
+  # @author qitang@redhat.com
+  # @case_id OCP-21389
+  @admin @destructive
+  Scenario: Deploy Openshift Event following Docs
+    Given I switch to the first user
```

Comment:
Unnecessary step ?

Reply:
I think no, because when I deploy logging resources, I have to use the cluster-admin user, so here if I don't switch user, the user would be a cluster-admin user.